### PR TITLE
Blog Cassandra Changelog #2

### PR DESCRIPTION
**Pull request by loganloganlogan**
New blog for the monthly Changelog series.

### CODE REVIEW

File: src/_posts/2020-09-17-cassandra-changelog-2 [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## Community Notes
+
+_Updates on Cassandra Enhancement Proposals (CEPs), how to contribute, and other community activities._
+
+#### **Added**
+The community weighed options to address reads inconsistencies for Compact Storage as noted in ticket [CASSANDRA-16217](https://issues.apache.org/jira/browse/CASSANDRA-16217) (committed). The conversation continues in ticket [CASSANDRA-16226](https://issues.apache.org/jira/browse/CASSANDRA-16226) with the aim of ensuring there are no huge performance regressions for common queries when you upgrade from 2.x to 3.0 with Compact Storage tables or drop it from a table on 3.0+.
+
+#### **Added**
+In 4.0 there is a plan to drop compact storage and related code, but the community noticed behavioral changes such as differences in visible behavior between "normal" and compact tables. Solutions are being discussed as part of [CASSANDRA-16217](https://issues.apache.org/jira/browse/CASSANDRA-16217).
```
Comment (michaelsembwever) [inline src/_posts/2020-09-17-cassandra-changelog-2:30]: I think this item and the one above (line 27) are the same thing…?

Comment (loganloganlogan) [reply src/_posts/2020-09-17-cassandra-changelog-2:30]: Ah yes, thank you! One should have been deleted - whichever one sounds better please delete the other.

Comment (loganloganlogan) [reply src/_posts/2020-09-17-cassandra-changelog-2:30]: @michaelsembwever Shall I remove and resubmit?

**Comment by michaelsembwever** [comment]
superseded by https://github.com/apache/cassandra-website/pull/29

**Pull request closed without merging**